src/demos/clash/HealthBar.cs

```diff
@@ -0,0 +1,35 @@
+using Godot;
+using System;
+
+public partial class HealthBar : Node2D
+{
+    [Export]
+    Health health;
+
+    [Export]
+    Node2D healthSprite;
+
+    private Node2D currentNode;
+    private float startingHealth;
+
+    // Called when the node enters the scene tree for the first time.
+    public override void _Ready()
+    {
+        currentNode = GetNode<Node2D>("./");
```

Comment:
You don't need this because `HealthBar` is already of type `Node2D`. Use `this` instead

Reply:
I've removed the redundant call.